framework/components/ema-rating-form/ema-rating-form.config.js

```diff
@@ -1,13 +1,24 @@
+const feedback = [
+  'Unhelpful',
+  'Not very helpful',
+  'Fairly helpful',
+  'Helpful',
+  'Very helpful',
+];
+const selector = 'ema-rating-1';
+
 module.exports = {
   title: 'Rating form',
   label: 'Rating form',
   status: 'ready',
   context: {
     _demo: {
       scripts: `document.addEventListener('DOMContentLoaded', function () {
-        ECL.ratingForm();
+        ECL.ratingForm(${JSON.stringify(selector)});
+        ECL.ratings(${JSON.stringify(feedback)});
```

Comment:
Isn't is possible to call `ECL.ratings()` in `ECL.ratingForm()`? So that developers only have one script to call.
Also is is mandatory to use `JSON.stringify`? for instance, selector is already a string.

Reply:
It's eiter `ECL.ratingForm(${JSON.stringify(selector)});` or `ECL.ratingForm("${selector}");`